swagger/README.md

```diff
@@ -0,0 +1,19 @@
+# Tekton dashboard REST API - Open API
+
+The API definitions are in tekton-dashboard-api.yaml file
```

Comment:
We can include a hyperlink in the readme to the actual file reference

Reply:
I added it.